lib/KCard/index.vue

```diff
@@ -167,6 +172,19 @@
         type: String,
         default: 'centerInside',
       },
+      /**
+       * Controls the alignment of the thumbnail area in horizontal layouts.
+       * Only applies to horizontal layouts with 'small' or 'large' thumbnail display.
+       * Ignored in other layouts.
+       * @type {String}
+       * @values 'left', 'right'
+       * @default 'left'
+       */
+      thumbnailAlign: {
+        type: String,
+        default: thumbnailAlign.LEFT,
```

Comment:
Needs to be `default: 'left'` because the documentation build won't otherwise show the correct default value.